ssm_jax/hmm/demos/low-rank_gaussian_hmm.ipynb

```diff
@@ -0,0 +1,275 @@
+{
```

Comment:
<p>I think it's important to say "<em>diagonal plus </em>low rank." If the covariance were truly low rank, the Gaussian density would be degenerate.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/ssm-jax/pull/157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I've changed this throughout the text, but are you OK if the import is still `LowRankHMM` ? Or do you have another suggestion that's not two abbreviations (i.e. DPLR_HMM) ? 😅